dependencies.yml

```diff
@@ -49,9 +49,9 @@ com.fasterxml.jackson.core:
 
 com.github.ben-manes.caffeine:
   caffeine:
-    version: '2.8.8'
+    version: '3.0.1'
```

Comment:
How about upgading this when we upgrade it in Armeria? 😄

Reply:
Makes sense. 👍